docs/encrypt_decryt_configuration.md

```diff
@@ -98,3 +100,68 @@ SQLALCHEMY_DATABASE_URI: mysql+mysqlconnector://user_of_db:user_of_db@localhost/
 ```
 
 And you can access to this var with `current_app.config["SQLALCHEMY_DATABASE_URI"]`
+
+## 1. Encrypt your information with AWS KMS and base64
+
+Pyms knows if a variable is encrypted if this var start with the prefix `enc_` or `ENC_`. PyMS uses boto3 and
+aws cli to decrypt this value and store it in the same variable without the `enc_` prefix.
+
+First, configure aws your aws account credentials:
+
+```bash
+aws configure
+```
+
+Cypher a string with this command:
+
+```bash
+aws kms encrypt --key-id alias/your-kms-key --plaintext "mysql+mysqlconnector://important_user:****@localhost/my_schema" --query CiphertextBlob --output text | base64 -d | base64
```

Comment:
What this line does is decode the bas64 and then reencode, doesn't it? Maybe the last two pipes are unnecessary? @avara1986

Reply:
Also, it seems to me that both examples are in base64.